holidays/countries/united_kingdom.py

```diff
@@ -109,6 +128,10 @@ def _populate_subdiv_eng_public_holidays(self):
         # Easter Monday
         self._add_easter_monday("Easter Monday")
 
+        # Whit Monday.
```

Comment:
Better to move it to `_populate_subdiv_holidays()`, since there is already a group of holidays that are not exist in Scotland only. Perhaps even together with Easter Monday and Late Summer Bank Holiday, as they have identical rules in ENG, NIR and WLS.

Reply:
That will affect Isle of Man too, in which I haven't found any evidence yet if they ever have this in their holiday list pre-1971 os that's something we need to keep in mind.

Reply:
The only evidence Isle of Man ever celebrates Whit Monday at all is this [British Pathe video](https://www.youtube.com/watch?app=desktop&v=JzdJ7v93A6M) - not exactly perfect but this will do for now. :+1:

Reply:
Sorry, my fault - I forgot that this code is also working for whole UK now (without `subdiv`), so we can't place Whit Monday here (as there should be only holidays that exist in all provinces).

Reply:
After spending the whole morning looking for Manx pre-1971 calendar, I can now confirm that Isle of Man does indeed have Whit Monday as public holidays at one point (or at least for 1968): https://namanx.org/holy-days-a-hanging-buried-treasure-all-in-a-manx-calendar

![image](https://github.com/vacanza/python-holidays/assets/19505219/4f2276ef-ba31-4454-89dc-1a9188c0ef7d)
![image](https://github.com/vacanza/python-holidays/assets/19505219/d1af1c21-2915-42d6-8aba-b78423dc144f)
![image](https://github.com/vacanza/python-holidays/assets/19505219/7c79e6b9-e32e-4f8f-8a29-4f89ca18499e)

Reply:
You're really digging to deep, it's just fantastic! 👍 But I think we need to move Whit Monday from `_populate_subdiv_holidays()` to specific subdivisions (and the Isle of Man), as you did at first (because now we have Whit Monday in whole UK as well, which is incorrect). I'm sorry for misleading you with [that comment](https://github.com/vacanza/python-holidays/pull/1668#discussion_r1467545251).

Reply:
Fixed.